### DISCUSSION TITLE
What would be the reasons for a repo suddenly get cloned by a huge amount of times in a short period of time?

### DISCUSSION THREAD

**Discussion by ooker777** [community/community#54521] [Repositories] [answered]
What would be the reasons for why a repo suddenly get cloned by a huge amount of times in a short period of time? If that's a bot then what can it do with it? Unlike a referral spam, GitHub doesn't show anything about the cloners, so I guess this is not about marketing. They don't have push access.

About the data, well, it's open in GitHub. Anyone can use it. But why would they need to clone that much?

![](https://i.imgur.com/XAVMxR0.png)

**Comment by airtower-luna** [comment, marked as answer]
Do you have any CI workflows or the like that clone your repository (e.g. using `actions/checkout`)?

**Reply by byrneh** [reply]
Intriguing as on the 1/5. it says **52** uniqie clones and **655** clones.

**Reply by ooker777** [reply]
ah I see. Yeah I have. Is there other workflows that clone repos?

**Reply by airtower-luna** [reply]
I guess you mean other actions? Well, possibly. `actions/checkout` doesn't do anything another action couldn't do with `git clone`, and a composite action might even call it directly.

**Reply by ooker777** [reply]
do you know why GH doesn't differentiate human and bot clones?

**Reply by airtower-luna** [reply]
Nope. My best guess it that it adds complexity for very little benefit.
